README.md

```diff
@@ -1,2 +1,109 @@
 # robo-joomla
-Robo tasks for Joomla! development, build, testing and everything  
+Robo tasks for Joomla! development, build, testing and everything
+
+
+## Installation (Standalone):
+
+  * `composer install`
+  * Initialized here. Please run `robo init` to create a new RoboFile.
```

Comment:
should this be `vendor/bin/robo init`, but then I get this error: Command "init" is not defined.

Reply:
@810 Thank you very much for testing. 
Did you get an error when you try <tt>composer install</tt>?
In my case composer install worked. I have got installed it global.

Reply:
![robo](https://user-images.githubusercontent.com/876623/38133538-3715125c-33ff-11e8-9fb1-19e0a33c2d4c.jpg)

Reply:
@810 Now I saw the place where I forgot the <tt>vendor/bin</tt> I changed it to <tt>vendor/bin/robo init</tt>. Thank you.

Which branch are you on? <tt>vendor/bin/robo init</tt> only works in <tt>develop</tt> or my <tt>readme</tt> branch.

Reply:
i used your branch astridx:readme

Reply:
Did you use 
<tt>vendor/bin/robo init</tt>
On my branch aswell? 
In the picture I see only 
<tt>robo init</tt> (my first mistake)

Reply:
yes, see command above that.